openpower/patches/vesnin-patches/skiboot/skiboot-1001-core-direct-controls-Add-sreset-support-for-P8.patch

```diff
@@ -0,0 +1,52 @@
+From 81b30b55eea07d84a4e9d84e73495a5a67f83ecf Mon Sep 17 00:00:00 2001
+From: Maxim Polyakov <m.polyakov@yadro.com>
+Date: Fri, 18 Jan 2019 17:49:37 +0300
+Subject: [PATCH] core/direct-controls: Add sreset support for P8
+
+VESNIN server uses NMI SRESET signal to initiate memory dumping with
+kdump in case of crash of the host OS. It is not necessary to return
+control to the host's kernel since the Linux has been already
+crashed. This commit permits registration and handling
```

Comment:
permits registration and handling => allows for registration and handling of

Reply:
Fixed